cognite/client/_api/simulators/runs.py

```diff
@@ -0,0 +1,218 @@
+        routine_external_ids: Sequence[str] | None = None,
+        routine_revision_external_ids: Sequence[str] | None = None,
+        model_revision_external_ids: Sequence[str] | None = None,
+    ) -> SimulatorRunsList:
+        """`Filter simulation runs <https://developer.cognite.com/api#tag/Simulation-Runs/operation/filter_simulation_runs_simulators_runs_list_post>`_
+        Retrieves a list of simulation runs that match the given criteria
+
+        Args:
+            limit (int | None): The maximum number of simulation runs to return, pass None to return all.
+            status (str | None): Filter by simulation run status
+            run_type (str | None): Filter by simulation run type
+            model_external_ids (Sequence[str] | None): Filter by simulator model external ids
+            simulator_integration_external_ids (Sequence[str] | None): Filter by simulator integration external ids
+            simulator_external_ids (Sequence[str] | None): Filter by simulator external ids
+            routine_external_ids (Sequence[str] | None): Filter by routine external ids
+            routine_revision_external_ids (Sequence[str] | None): Filter by routine revision external ids
+            model_revision_external_ids (Sequence[str] | None): Filter by model revision external ids
+
+        Returns:
+            SimulatorRunsList: List of simulation runs
+
+        Examples:
+
+            List simulation runs:
+                >>> from cognite.client import CogniteClient
+                >>> client = CogniteClient()
+                >>> res = client.simulators.runs.list()
+
+            Filter runs by status and simulator external ids:
+                >>> res = client.simulators.runs.list(
+                ...     simulator_external_ids=["PROSPER", "DWSIM"],
+                ...     status="success"
+                ... )
+        """
+
+        filter_runs = SimulatorRunsFilter(
+            status=status,
+            run_type=run_type,
+            model_external_ids=model_external_ids,
+            simulator_integration_external_ids=simulator_integration_external_ids,
+            simulator_external_ids=simulator_external_ids,
+            routine_external_ids=routine_external_ids,
+            routine_revision_external_ids=routine_revision_external_ids,
+            model_revision_external_ids=model_revision_external_ids,
+        )
+        self._warning.warn()
+        return self._list(
+            method="POST",
+            limit=limit,
+            resource_cls=SimulationRun,
+            list_cls=SimulatorRunsList,
+            filter=filter_runs.dump(),
+        )
+
+    @overload
+    def create(self, run: SimulationRunWrite) -> SimulationRun: ...
+
+    @overload
+    def create(self, run: Sequence[SimulationRunWrite]) -> SimulatorRunsList: ...
+
+    def create(self, run: SimulationRunWrite | Sequence[SimulationRunWrite]) -> SimulationRun | SimulatorRunsList:
```

Comment:
How to you plan to add support for create by routine reversion. It seems you only have by routine now
![image](https://github.com/user-attachments/assets/0c0d01a7-73d2-4230-a41c-a01f06f66919)

Reply:
since it's a feature, can it be a separate PR? we are over 500 limit already